java/HealthVault Mobile/HVSample/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,237 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+	xmlns:design="http://schemas.android.com/apk/res-auto"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+	android:background="#e3e3e3"
+    android:orientation="vertical"
+    tools:context=".MainActivity">
+
+    <LinearLayout
```

Comment:
We have tons of copied dimensions and colors in this layout. Take advantage of dimens.xml and colors.xml to clean these up and reference a root value instead.